README.md

```diff
@@ -37,13 +37,13 @@ The Gravity Bridge requires some additional pieces to be deployed to support it:
 mkdir install && cd install
 
 # Install Orchestrator
-wget https://github.com/PeggyJV/gravity-bridge/releases/download/v0.1.21/contract-deployer https://github.com/PeggyJV/gravity-bridge/releases/download/v0.1.21/orchestrator https://github.com/PeggyJV/gravity-bridge/releases/download/v0.1.21/relayer https://github.com/PeggyJV/gravity-bridge/releases/download/v0.1.21/gorc && chmod +x * && sudo mv * /usr/bin
+wget https://github.com/PeggyJV/gravity-bridge/releases/download/v0.3.8/contract-deployer https://github.com/PeggyJV/gravity-bridge/releases/download/v0.3.8/orchestrator https://github.com/PeggyJV/gravity-bridge/releases/download/v0.3.8/relayer https://github.com/PeggyJV/gravity-bridge/releases/download/v0.3.8/gorc && chmod +x * && sudo mv * /usr/bin
```

Comment:
Only the gorc binary should be necessary for the orchestrator now, so we can delete the other `wget` commands and update the version to `0.3.9`.

Reply:
Corrected